src/lib/pay-request/services/connector/types.ts

```diff
@@ -210,3 +210,8 @@ export interface AddGatewayAccountCredentialsRequest {
   payment_provider: string;
   credentials?: StripeCredentials;
 }
+
+export interface AddGithubAndZendeskCredential {
```

Comment:
I’m going to be picky here and point out it’s actually ‘GitHub’ (but it is ‘Zendesk’!).